lib/app/router/main_tabs/components/profile_tab_button.dart

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/components/avatar/avatar.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/app/features/user/providers/user_metadata_provider.dart';
+
+class ProfileTabButton extends ConsumerWidget {
+  const ProfileTabButton({
+    required this.isSelected,
+    super.key,
+  });
+
+  static double get avatarSize => 13.0.s;
+
+  double get borderWidth => 1.2.s;
+
+  final bool isSelected;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final userPicture = ref.watch(currentUserMetadataProvider).valueOrNull?.data.picture;
```

Comment:
wdyt about selecting the picture object so that instead of re-rendering this widget whenever currentUserMetadataProvider changes, we render it only when the picture changes?